### PR TITLE
Update Android plugin and make buildtools explicit

### PR DESCRIPTION
**Pull request by ahmedre**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -6,6 +6,7 @@ apply plugin: 'kotlin-kapt'
 
 android {
   compileSdkVersion deps.android.build.compileSdkVersion
+  buildToolsVersion '28.0.2'
```
Comment (ozbek) [inline app/build.gradle]: Why is this needed?

Comment (ahmedre) [reply app/build.gradle]: without it, it's defaulting to this version anyway (forced by the Android gradle plugin i think?) - so figured might as well set it explicitly so it's not resolving to something different without us knowing what it is.

Comment (ozbek) [reply app/build.gradle]: That's right, the tooling version is selected automatically depending on the Android Gradle version. [The prior change](https://github.com/quran/quran_android/pull/946) was intentional :)

**Comment by ahmedre** [comment]
abandoning in favor of #978
